test/jarohen/with_open_test.clj

```diff
@@ -0,0 +1,57 @@
+(ns jarohen.with-open-test
+  (:require
+   [clojure.test :refer [deftest is]]
+   [jarohen.with-open :as with-open]
+   [matcher-combinators.test]))
```

Comment:
We don't have CI for this, but I think this test will fail when you run them from the command line. Matchers isn't in `deps.edn`. I don't see any of the matchers used, so maybe it just needs to be removed.

Reply:
Pretty sure that the dep is there on line 7 in deps.edn, but you're right that this appears unused. I'll see if tests still work without this require.

Reply:
What I meant to say is you have an unused require :laughing: